scripts/dictionary.txt

```diff
@@ -201,10 +207,21 @@ unformatted
 Unordered
 unparsed
 Unsplash
+Utils
 vars
 vimrc
 VPC
+Vue
+<<<<<<< HEAD
```

Comment:
@ShawnHosea it looks like there was a merge conflict here.
Needs to have the conflict fixed and the conflict comments `<<<` etc removed